docs/doxygen/Doxyfile

```diff
@@ -969,6 +969,8 @@ INPUT                  = ../../README.md \
                         ../../include/rppt_tensor_geometric_augmentations.h \
                         ../../include/rppt_tensor_morphological_operations.h \
                         ../../include/rppt_tensor_statistical_operations.h \
+                        ../../include/rppt_tensor_arithmetic_operations.h \
+                        ../../include/rppt_tensor_audio_augmentations.h \
```

Comment:
I would list these in alphabetical order just for grins.

Reply:
yup, changed